tinker/templates/faculty-bios/form.html

```diff
@@ -79,6 +79,19 @@ <h3 id="{{ field.id }}">{{ field.label }}</h3>
                             {% elif field.type == "CKEditorTextAreaField" %}
                                 {{ field(class="ckeditor") }}
 
+                            {% elif field.type == 'RadioField' %}
+                                {% if show_special_admin_view %}
```

Comment:
i mentioned it earlier, but this check should be done in forms.py

Reply:
The hidden field should actually handle the value nicely - which means it won't even use this if section "field.type == 'RadioField'"

Reply:
fixed